src/software/ai/hl/stp/stp_refbox_game_state_play_selection_test.cpp

```diff
@@ -34,11 +34,11 @@ class STPRefboxGameStatePlaySelectionTestWithPositions
    protected:
     void SetUp() override
     {
-        world.mutableField() = ::Test::TestUtil::createSSLDivBField();
+        world.updateField(::Test::TestUtil::createSSLDivBField());
```

Comment:
I think `setField` might make more sense? `update` suggests the potential for a partial change of some type (at least to me)

Reply:
removed it completely since it was only used in tests